app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -306,6 +326,61 @@ class StoryFragmentTest {
     }
   }
 
+  @Test
+  fun testStoryFragment_chapterMissingPrerequisiteThumbnailIsBlurred() {
+    launch<StoryActivity>(createFractionsStoryActivityIntent()).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(allOf(withId(R.id.story_chapter_list))).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          2
+        )
+      )
+
+      onView(
```

Comment:
Can we try putting delay here so that it will wait till image loaded, using test routine dispatchers.advanceUntilIdle.

Reply:
@anandwana001 I tried, nothing changed. Test still passes.

Reply:
Done.